lib/language.test.js

```diff
@@ -73,31 +77,33 @@ describe('languageHandler', () => {
 
   it('should use language cookie if requestedLanguage is invalid', () => {
```

Comment:
What do we want to test here? It doesn't look like we have a language cookie, so that isn't the one that ends up being set